src/test/shell/bazel/remote/remote_execution_http_test.sh

```diff
@@ -26,15 +26,15 @@ function set_up() {
   work_path=$(mktemp -d "${TEST_TMPDIR}/remote.XXXXXXXX")
   pid_file=$(mktemp -u "${TEST_TMPDIR}/remote.XXXXXXXX")
   attempts=1
-  while [ $attempts -le 5 ]; do
+  while [ $attempts -le 1 ]; do
```

Comment:
I think the number of attempts here were because of port collision.

Reply:
+1

Reply:
Ah, sorry! I forgot to revert this after debugging my new code. Fixed.